gather_istio

```diff
@@ -17,6 +17,10 @@
 MUST_GATHER=${MUST_GATHER:-/usr/bin/openshift-must-gather}
 BASE_COLLECTION_PATH="/must-gather"
 
+# The following CRDs are not managed by the Maistra operator but it
+# will create instances of them.
+DEPENDENCY_CRDS="jaegers.jaegertracing.io kialis.kiali.io"
```

Comment:
I'd call this just `CR`, not `CRD`, to avoid confusion.

Reply:
I agree, I changed it